src/sklearn_evaluation/plot/clustering.py

```diff
@@ -89,27 +89,23 @@ def elbow_curve(
 
     Notes
     -----
-    .. deprecated:: 0.8.7
-        'n_clusters' renamed to 'range_n_clusters' and will be removed in version 0.8.9
+    .. deprecated:: 0.9
```

Comment:
not .8.7?

Reply:
since we're applying a pending deprecation the next release must be a major one. so it wont be 0.8.7, it'll be 0.9